index.ts

```diff
@@ -1772,6 +1772,65 @@ const http1Page = async (
   } as unknown as ClientHttp2Session;
 };
 
+const workerPage = (proxyHostnameAndPort: string, state: State) => {
+  return staticResponse(
+    `
+const mapping = ${JSON.stringify(
+      Object.entries(state.config.mapping)
+        .filter(key => key && key.length > 1)
+        .map(([key, value]) => {
+          if (typeof value === "string" && value.startsWith("data:"))
+            return [key, "data:text/plain,..."];
+          let match: RegExpMatchArray | null =
+            key.match(RegExp(key.replace(/^\//, "^/"))) ?? null;
+          const replacedReplaceBody = (
+            typeof value === "string" ? value : value.replaceBody
+          )?.replace(
+            /\$\$(\d+)/g,
+            (_, index) => match?.[parseInt(index)] ?? "",
+          );
+          let replacementCounter = 0;
+          return [
+            key.replace(/\([^)]+\)/g, () => `$${++replacementCounter}`),
```

Comment:
post increment makes more sense since it would start at 0

Reply:
It starts with 1 actually (to imitate how regex replacements work),
see https://github.com/libetl/local-traffic/pull/349